recipes/gulp/gulpfile.js

```diff
@@ -57,7 +57,8 @@ const runPwmetrics = function() {
  */
 const handleOk = function(results) {
   stopServer();
-  return results;
+  console.log(results);
+  process.exit(0);
```

Comment:
yeap, because travis will not be closed. I noticed that during build https://travis-ci.org/denar90/pwmetrics/jobs/221887465
And we wanna pass our recipe which is also like integration test